src/pages/progress_reports/SubmitProgressReport.tsx

```diff
@@ -5,35 +5,125 @@ import {Notifications} from 'src/libs/utils';
+      expectedPublication.datasetCitation = pub.dataset_citation;
+      expectedPublication.citation = pub.did_cite;
+      return expectedPublication;
+    });
+  };
+
+  const getPresentationList = (formState: FormState): Presentation[] => {
+    const presentations: PublicationOrPresentation[] = formState.presentations ?? [];
+    return presentations.map((pub: PublicationOrPresentation) => {
+      const expectedPresentation: Presentation= {} as Presentation;
+      expectedPresentation.title = pub.title;
+      expectedPresentation.date = pub.date;
+      expectedPresentation.authors = pub.authors;
+      expectedPresentation.datasetCitation = pub.dataset_citation;
+      expectedPresentation.citation = pub.did_cite;
+      expectedPresentation.link = pub.link;
+      return expectedPresentation;
+    });
+  };
+
+  const getDataManagementIncidents = (formState: FormState): DataManagementIncident => {
+    const dataManagementIncident: DataManagementIncident = {} as DataManagementIncident;
+    dataManagementIncident.incidents = []
+    DMI_INCIDENT_KEYS.map((key) => {
+      const incident = formState[key] ?? undefined;
+      if (incident) {
+        dataManagementIncident.incidents.push(key);
+      }
+    });
+    dataManagementIncident.description = formState.dmiDescription ?? '';
+
+    return dataManagementIncident;
+  }
+
+  const convertFormStateToDAR = (formState: FormState): Partial<DataAccessRequest> => {
+    const expectedForm: Partial<DataAccessRequest> = {} as Partial<DataAccessRequest>;
+    expectedForm.progressReportSummary = formState.progressReportSummary;
+    if (formState.intellectualPropertyYesNo) {
+      expectedForm.intellectualPropertySummary = formState.intellectualPropertySummary;
+    }
+    expectedForm.datasetIds = formState.datasetIds ?? [];
+    if (formState.publicationsYesNo) {
+      expectedForm.publications = getPublicationList(formState);
+    }
+    if (formState.presentationsYesNo) {
+      expectedForm.presentations = getPresentationList(formState);
+    }
+    expectedForm.labCollaborators = formState.labCollaborators;
+    expectedForm.internalCollaborators = formState.internalCollaborators;
+    expectedForm.externalCollaborators = formState.externalCollaborators;
+    if (formState.dmiYesNo) {
+      expectedForm.dmi = getDataManagementIncidents(formState);
+    }
+    if (formState.closeoutYesNo) {
+      expectedForm.closeoutSupplement = formState.closeoutSupplement;
+    }
+    return expectedForm;
+  }
 
   const submit = async () => {
     if (validationFailed(validateForm())) {
       Notifications.showError({text: "Form validation failed. Please check the form for errors."});
     } else {
       try {
-        const submittedPR = await ProgressReport.submitProgressReport(progressReport, parentReferenceId);
+        const submittedPR = await ProgressReport.submitProgressReport(createMultiPartFormData(convertFormStateToDAR(formState)), parentReferenceId);
```

Comment:
```suggestion
        const multiPartFormData = createMultiPartFormData(convertFormStateToDAR(formState));
        const submittedPR = await ProgressReport.submitProgressReport(multiPartFormData, parentReferenceId);
```

Small change for clarity.